week2/problem2.5.9.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+//Given a string, if the string begins with "red" or "blue" return that color string, otherwise return the empty string.
+
+func seeColor(givenString string) {
+	if strings.Contains(givenString[0:3], "red") {
```

Comment:
why you use [0:3], [0:4]?

Reply:
searching for "red" or "blue" substring in the beggining of the given string. 
could be: 

substring1 := "red"
subtring2 := "blue"
if strings.Contains(givenString[0 : len(substring1], substring1) { ... .

Though, the problem descrition never specified that the searched substrings could change.